ai_summary/indico_ai_summary/client/components/ind_summarize_button.jsx

```diff
@@ -0,0 +1,301 @@
+// This file is part of the Indico plugins.
+// Copyright (C) 2002 - 2025 CERN
+//
+// The Indico plugins are free software; you can redistribute
+// them and/or modify them under the terms of the MIT License;
+// see the LICENSE file for more details.
+
+// Main entry point of the summarization feature in the Indico plugin.
+// This file defines the Summarize button, manages modal states,
+// handles prompt selection, generates summaries, and integrates
+// with Indico's API to fetch and save meeting notes.
+
+// import getStoredPrompts from 'indico-url:plugin_ai_summary.stored_prompts';
```

Comment:
not used, but maybe getting it here would be nicer since it avoids querying it on every page load by a manager...

with `useIndicoAxios` it's quite easy to do this